src/Propulsion.Cosmos/PropulsionCosmosPrometheus.fs

```diff
@@ -1,114 +1,147 @@
+/// This file implements a Serilog Sink `LogSink` that publishes metric values to Prometheus.
+/// It takes in an additional set of custom tags to annotate the metric we're publishing.
```

Comment:
that is not a core feature of the thing - I'd add that as a good `///` comment on the tags arg